spk/itools/Makefile

```diff
@@ -0,0 +1,28 @@
+SPK_NAME = itools
+SPK_VERS = 1.0
+SPK_REV = 1
+SPK_ICON = src/itools.png
+
+DEPENDS = cross/usbmuxd cross/ifuse
+UNSUPPORTED_ARCHS = $(ARM_ARCHES) $(PPC_ARCHES)
+
+MAINTAINER = BingJing
+MAINTAINER_URL = https://www.linkedin.com/in/bingjing-chang/
+DISTRIBUTOR = SynoCommunity
+DISTRIBUTOR_URL = https://synocommunity.com
+DESCRIPTION = A collection toolkit for connecting iOS devices.
+RELOAD_UI = n
+DISPLAY_NAME = iTools
+STARTABLE = no
+BETA = 1
+CHANGELOG = First Release
+
+HOMEPAGE   = https://www.libimobiledevice.org
+LICENSE    = LGPL 2.1
+
+INSTALLER_SCRIPT = src/installer.sh
+SSS_SCRIPT       = src/dsm-control.sh
```

Comment:
Simply replace by `SERVICE_COMMAND = $${SYNOPKG_PKGDEST}/bin/usbmuxd`
If that deamon generates a pid file, please add option to use variable `PID_FILE`

Reply:
Question to answer, is that service only runs properly as `root` or is it expected to work properly as non privileged user?

Reply:
It doesn't generate a pid file; however, SVC_WRITE_PID=y is really useful. 
It should be run as root. It requires USB permission.